docs/contributing/index.md

```diff
@@ -1,310 +1,153 @@
 ---
 myst:
   html_meta:
-    "description": "Contributing to Plone 6 Documentation"
-    "property=og:description": "Contributing to Plone 6 Documentation"
-    "property=og:title": "Contributing to Plone 6 Documentation"
-    "keywords": "Plone, Plone Contributor Agreement, License, Code of Conduct"
+    "description": "Contributing to Plone Core Code"
+    "property=og:description": "Contributing to Plone Core Code"
+    "property=og:title": "Contributing to Plone Core Code"
+    "keywords": "Plone, Plone Contributor Agreement, License, Code of Conduct, core, code"
 ---
 
-(contributing-index-label)=
+(contributing-to-plone-core-code-index-label)=
 
-# Contributing to documentation
+# Contributing to Plone core code
 
-This document describes how to contribute to Plone Documentation.
+This part of the documentation describes how to contribute to Plone core code, including all its packages and repositories under the Plone GitHub organization.
 
-Contributions to the Plone Documentation are welcome.
+To contribute to any package in Plone, you must follow the policies of the [Plone Foundation](https://plone.org/foundation), [Plone GitHub organization](https://github.com/plone/) and the specific package.
 
+This chapter covers policies that apply to all Plone packages.
+Other chapters cover any variations and additional policies for each package.
 
-(contributing-permission-to-publish-label)=
 
-## Granting permission to publish
+(contributing-sign-and-return-the-plone-contributor-agreement-label)=
 
-Before you contribute, you must give permission to publish your contribution according to the license we use.
-You may give that permission in two ways.
+## Sign and return the Plone Contributor Agreement
 
-- Sign the [Plone Contributor Agreement](https://plone.org/foundation/contributors-agreement).
-  This method also covers contributions to Plone code.
-  It is a one-time only process.
-- In every pull request or commit message, include the following statement.
+You must give permission to the Plone Foundation to publish your contribution, according to the license we use.
+Plone uses the [GNU General Public License, version 2](https://github.com/plone/Products.CMFPlone/blob/master/LICENSE).
```

Comment:
This only partially true, some code is BSD and some ZPL.

Reply:
Please provide specifics so we can provide explicit instructions.

For the Plone Foundation and for repos under the GitHub Plone organization, which repos do not use the GPLv2?

Reply:
I revised the language in https://github.com/plone/documentation/pull/1478/commits/caf5a0b3ad1c18733247fe5570b4479e3476f6f6. I don't want users to get lost in the licensing weeds, as we want them to sign the PCA.

I also created a new issue to update broken links: https://github.com/plone/plone.org/issues/112